docs/snippets/install-kn.md

```diff
@@ -0,0 +1,50 @@
+!!! todo "Installing the `kn` CLI"
+
+    === "Using Homebrew"
+        For macOS, you can install `kn` by using <a href="https://github.com/knative/homebrew-client" target="_blank">Homebrew</a>.
+
+        ```
+        brew install knative/client/kn
+        ```
+
+    === "Using a binary"
+
+        You can install `kn` by downloading the executable binary for your system and placing it in the system path.
+
+        A link to the latest stable binary release is available on the <a href="https://github.com/knative/client/releases" target="_blank">`kn` release page</a>.
+
+    === "Using Go"
+
+        1. Check out the `kn` client repository:
+
+              ```
+              git clone https://github.com/knative/client.git
+              cd client/
+              ```
+
+        1. Build an executable binary:
+
+              ```
+              hack/build.sh -f
+              ```
+
+        1. Move `kn` into your system path, and verify that `kn` commands are working properly. For example:
+
+              ```
+              kn version
+              ```
+
+    === "Using a container image"
+
+        Links to images are available here:
+
+        - <a href="https://gcr.io/knative-releases/knative.dev/client/cmd/kn" target="_blank">Latest release</a>
+
+        You can run `kn` from a container image. For example:
+
+        ```
+        docker run --rm -v "$HOME/.kube/config:/root/.kube/config" gcr.io/knative-releases/knative.dev/client/cmd/kn:latest service list
+        ```
+
+        !!! note
```

Comment:
One super minor nit but I'd take this out of the note and put it before the numbered steps in this bit, just cos it's important and also a note inside a tab like this looks kinda odd. Your call though.

Reply:
This isn't a blocker so I'll let @julz lgtm cos it looks like he had suggestions also